lefthook.yml

```diff
@@ -0,0 +1,39 @@
+pre-commit:
+  parallel: true
+  jobs:
+    - name: Format non-markdown
+      glob: "*.{js,json,yml,yaml,css,html}"
+      run: yarn prettier --write --cache {staged_files}
+      stage_fixed: true
+
+    - name: Fix markdown
+      glob: "*.md"
+      group:
+        parallel: true
+        jobs:
+          - name: Markdownlint
+            run: yarn markdownlint-cli2 --fix {staged_files}
+            stage_fixed: true
+          - name: Check URLs per locale
+            run: yarn node ./scripts/check-url-locale.js --fix {staged_files}
+            stage_fixed: true
+          - name: Autocorrect
+            run: yarn autocorrect --fix {staged_files}
+            stage_fixed: true
+          - name: Check changed xrefs
+            run: yarn node scripts/update-moved-file-links.js --check
```

Comment:
I'm not sure if this is correct, because there is no such script file in translated-content

Reply:
Should be fixed in 74c1ffad7d5ed44e07f4101e037afe83d6370d3a